ext/sockets/sockets.c

```diff
@@ -1971,15 +1990,15 @@ PHP_FUNCTION(socket_set_option)
 
 #ifdef SO_ATTACH_REUSEPORT_CBPF
 		case SO_ATTACH_REUSEPORT_CBPF: {
-			convert_to_long(arg4);
+			zend_long fval = zval_get_long(arg4);
```

Comment:
`fval` ? what does the `f` stand for?

Reply:
f for function (for CBPF) val.

Reply:
I think I would prefer `cbpf_val` then, as when I see `fval` I'm thinking "float val".